cli.js

```diff
@@ -3,14 +3,17 @@
 var meow = require('meow');
 var killTabs = require('./');
 
-meow({
+var cli = meow({
 	help: [
-		'Usage',
-		'  $ kill-tabs'
+		'Usage: kill-tabs [OPTIONS]',
+		'  --no-chromium       Do not kill tabs from chromium',
+		'  --chromium          Kill tabs from chromium (default)',
+		'  --no-chrome         Do not kill tabs from chrome',
+		'  --chrome            Kill tabs from chrome (default)'
 	]
 });
 
-killTabs(function (err) {
+killTabs(cli, function (err) {
```

Comment:
You should pass in `cli.flags`, not the CLI. The API should have no knowledge of the CLI.